src/common.rs

```diff
@@ -981,9 +981,45 @@ pub fn get_current_pid() -> Result<Pid, &'static str> {
     inner()
 }
 
+/// MAC address for network interface
+#[derive(PartialEq, Eq, Copy, Clone, Debug)]
+pub struct MacAddr {
+    data: [u8; 6],
+}
+
+impl MacAddr {
+    pub const UNSPECIFIED: Self = MacAddr::new();
+
+    #[inline]
+    pub(crate) const fn new() -> Self {
+        Self { data: [0; 6] }
+    }
+
+    pub fn data(&self) -> &[u8; 6] {
```

Comment:
I think we should make the field public instead of having this method because otherwise, people can't modify the data.